spec/fixture/result.html

```diff
@@ -0,0 +1,167 @@
+---
+layout: testReport
+---
+                        
+                        <div class="total row">
+                        <h2 class="col-sm-18">Test Results</h2>
+                        <h2 class="col-sm-6 text-right">8 tests</h2>
+                    </div>
+                    <div>
+                    
+                        
+                            
+                            
+                        <div class="test-suite row test-suite--passing" onclick="changeDisplay('#test-1')">
+                            
+                            <h4 class="col-sm-20">FlyveMDMInventoryAgentUITests.FlyveMDMInventoryAgentUITests</h4>
+                            <h4 class="col-sm-4 text-right">1 tests</h4>
+                        </div>
+                        <!-- Failing or passing class -->
+                        <div id="test-1">
+                         
+                            
+                                
+                                    
+                                    
+                                <div class="test-case--passing row">
+                                 <div class="col-sm-20">
+                                     <p>testTakeScreenshots</p>
+                                 </div>
+                                 <div class="col-sm-4 text-right">
+                                     <p>26.674</p>
+                                 </div>
+                                </div>
+                                    
+                                
+                            
+                                
+                            
+                                
+                            
+                                
+                            
+                                
+                            
+                                
+                            
+                                
+                            
+                                
+                            
+                        </div>
+                         
+                            
+                            
+                        <div class="test-suite row test-suite--passing" onclick="changeDisplay('#test-2')">
+                            
+                            <h4 class="col-sm-20">FlyveMDMInventoryAgentTests.FlyveMDMInventoryAgentTests</h4>
+                            <h4 class="col-sm-4 text-right">7 tests</h4>
+                        </div>
+                        <!-- Failing or passing class -->
+                        <div id="test-2">
+                         
```

Comment:
Hi, 

This is because the file is generated like this, due to the if clause, also this file is to make the comparison for the final output of the tests

Reply:
Hi, @Naylin15 
Could you use this tool?
https://editorconfig.org/

Reply:
👍  I'll read the documentation